backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5663980_sys_gh13929_externalSystemConfig_Metasfresh.sql

```diff
@@ -0,0 +1,735 @@
+-- Column: ExternalSystem_Config_Metasfresh.ExternalSystem_Config_Metasfresh_ID
+-- 2022-11-07T12:55:39.792Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,707977,0,546666,613402,550001,'F',TO_TIMESTAMP('2022-11-07 14:55:39','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'ExternalSystem_Config_Metasfresh',10,0,0,TO_TIMESTAMP('2022-11-07 14:55:39','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- UI Element: Externes System(541024,de.metas.externalsystem) -> ExternalSystem_Config_Metasfresh(546666,de.metas.externalsystem) -> main -> 10 -> default.Suchschlüssel
+-- Column: ExternalSystem_Config_Metasfresh.ExternalSystemValue
+-- 2022-11-07T12:55:53.985Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,707980,0,546666,613403,550001,'F',TO_TIMESTAMP('2022-11-07 14:55:53','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'Suchschlüssel',20,0,0,TO_TIMESTAMP('2022-11-07 14:55:53','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- UI Element: Externes System(541024,de.metas.externalsystem) -> ExternalSystem_Config_Metasfresh(546666,de.metas.externalsystem) -> main -> 10 -> default.User Authentication Token
+-- Column: ExternalSystem_Config_Metasfresh.CamelHttpResourceAuthKey
+-- 2022-11-07T12:56:19.038Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,707978,0,546666,613404,550001,'F',TO_TIMESTAMP('2022-11-07 14:56:18','YYYY-MM-DD HH24:MI:SS'),100,'Y','Y','N','Y','N','N','N',0,'User Authentication Token ',30,0,0,TO_TIMESTAMP('2022-11-07 14:56:18','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-11-07T12:57:19.674Z
+/* DDL */ SELECT public.db_alter_table('ExternalSystem_Config_Metasfresh','ALTER TABLE public.ExternalSystem_Config_Metasfresh ADD COLUMN ExternalSystemValue VARCHAR(255) NOT NULL')
+;
+
+-- 2022-11-07T16:52:29.011Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Ref_List_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,543332,541255,TO_TIMESTAMP('2022-11-07 18:52:28','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Metasfresh',TO_TIMESTAMP('2022-11-07 18:52:28','YYYY-MM-DD HH24:MI:SS'),100,'Metasfresh','Metasfresh')
```

Comment:
- metasfresh ist spelled with a small "m".
- as written ealier today, without any further infos i don't know wtf this new type is about? is it aboutconnecting another metasfresh instance? must be, othwerwise "invoke metasfresh" as a process without any process description would not make sense, right?
- but i heard that "RabbitMQ REST API" is also about exchanging metasfresh stuff..if i as a user only knew what this all would be about